### PR TITLE
Fixed naming etc

### PR DESCRIPTION
**Pull request by santanusinha**

### CODE REVIEW

File: junit-testcontainer-aerospike/pom.xml [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
-    <artifactId>testcontainer-aerospike</artifactId>
+    <artifactId>junit-testcontainer-aerospike</artifactId>
     <version>1.0.4</version>
     <packaging>jar</packaging>
 
     <dependencies>
         <dependency>
-            <groupId>com.phonepe.testcontainer</groupId>
+            <groupId>io.appform.testcontainer</groupId>
             <artifactId>testcontainer-commons</artifactId>
```
Comment (jitendradhawan) [inline junit-testcontainer-aerospike/pom.xml]: rename to junit-testcontainer-commons

File: junit-testcontainer-demo/pom.xml [hunk 2]
```diff
@@ -5,25 +5,25 @@
...
-    <artifactId>testcontainer-demo</artifactId>
+    <artifactId>junit-testcontainer-demo</artifactId>
     <version>1.0.4</version>
     <packaging>jar</packaging>
 
     <dependencies>
         <dependency>
-            <groupId>com.phonepe.testcontainer</groupId>
+            <groupId>io.appform.testcontainer</groupId>
             <artifactId>testcontainer-commons</artifactId>
```
Comment (jitendradhawan) [inline junit-testcontainer-demo/pom.xml]: junit-testcontainer-commons

File: junit-testcontainer-demo/pom.xml [hunk 3]
```diff
@@ -5,25 +5,25 @@
...
             <version>1.0.4</version>
         </dependency>
         <dependency>
-            <groupId>com.phonepe.testcontainer</groupId>
+            <groupId>io.appform.testcontainer</groupId>
             <artifactId>testcontainer-aerospike</artifactId>
```
Comment (jitendradhawan) [inline junit-testcontainer-demo/pom.xml]: junit-testcontainer-aerospike

File: junit-testcontainer-elasticsearch/pom.xml [hunk 4]
```diff
@@ -6,18 +6,18 @@
...
-    <artifactId>testcontainer-elasticsearch</artifactId>
+    <artifactId>junit-testcontainer-elasticsearch</artifactId>
     <version>1.0.4</version>
     <packaging>jar</packaging>
 
     <dependencies>
         <dependency>
-            <groupId>com.phonepe.testcontainer</groupId>
+            <groupId>io.appform.testcontainer</groupId>
             <artifactId>testcontainer-commons</artifactId>
```
Comment (jitendradhawan) [inline junit-testcontainer-elasticsearch/pom.xml]: junit-testcontainer-commons

File: junit-testcontainer-rabbitmq/pom.xml [hunk 5]
```diff
@@ -5,18 +5,18 @@
...
-    <artifactId>testcontainer-rabbitmq</artifactId>
+    <artifactId>junit-testcontainer-rabbitmq</artifactId>
     <version>1.0.4</version>
     <packaging>jar</packaging>
 
     <dependencies>
         <dependency>
-            <groupId>com.phonepe.testcontainer</groupId>
+            <groupId>io.appform.testcontainer</groupId>
             <artifactId>testcontainer-commons</artifactId>
```
Comment (jitendradhawan) [inline junit-testcontainer-rabbitmq/pom.xml]: junit-testcontainer-commons

File: junit-testcontainer-aerospike/pom.xml (hunk 1 above)
Comment (santanusinha) [reply junit-testcontainer-aerospike/pom.xml]: Done

File: junit-testcontainer-demo/pom.xml (hunk 2 above)
Comment (santanusinha) [reply junit-testcontainer-demo/pom.xml]: Done

File: junit-testcontainer-demo/pom.xml (hunk 3 above)
Comment (santanusinha) [reply junit-testcontainer-demo/pom.xml]: Done

File: junit-testcontainer-elasticsearch/pom.xml (hunk 4 above)
Comment (santanusinha) [reply junit-testcontainer-elasticsearch/pom.xml]: Done

File: junit-testcontainer-rabbitmq/pom.xml (hunk 5 above)
Comment (santanusinha) [reply junit-testcontainer-rabbitmq/pom.xml]: Done

**Pull request merged**